doc: improve the multi-disk section on host page (#246)
harvester/docs: docs/host/host.md

1. Go to the **Hosts** page.
2. On the node you want to modify, click **⋮ > Edit Config**.

![Edit Config](/img/v1.1/host/edit-config.png)

3. Select the **Storage** tab and click **Add Disk**.

![Add Disks](/img/v1.1/host/add-disks.png)

:::caution

> Now, Harvester does not support adding specific partition as disk. Make sure your disk does not contain any partition or you could not find it at this stage.

Reviewer: Can u please help to double-check the deprecated version, thanks?

> Now, Harvester does not support adding specific partition as disk. Make sure your disk does not contain any partition or you could not find it at this stage.

Author: Sure, I would like to

> Now, Harvester does not support adding specific partition as disk. Make sure your disk does not contain any partition or you could not find it at this stage.

Author: I found the UI PR https://github.com/rancher/dashboard/pull/5297
That makes the user only use the whole disk.
And NDM also has some related PR for drop partition mountpoint.
So I thought that we do not support partition after v1.0.2.

> As of Harvester v1.0.2, we no longer support adding partitions as additional disks. If you want to add it as an additional disk, be sure to reformat it properly first(e.g., using `fdisk`).

Reviewer 2: ```suggestion
As of Harvester v1.0.2, we no longer support adding partitions as additional disks. If you want to add it as an additional disk, be sure to reformat it properly first (e.g., using `fdisk`).
```

Author: updated, thanks!

:::

4. Select an additional raw block device to add as an additional data volume.
    - The `Force Formatted` option is required if the block device has never been force-formatted.

![Force Format](/img/v1.1/host/force-format-disks.png)

> 5. Last, you can check the newly added disk by **⋮ > Edit Config** again. You can also add the NODE/DISK tag this time (Describe detail in the next section).

Reviewer 3: ```suggestion
5. Last, you can check the newly added disk by clicking **⋮ > Edit Config** again. You can also add the NODE/DISK tag this time (details are described in the next section).
```

Author: done, thanks!

> 5. Last, you can click **⋮ > Edit Config** again to check the newly added disk. At meanwhile, you can also add the "Host/Disk" tag (details are described in the [next section](#storage-tags)).

Reviewer 2: ```suggestion
5. Last, you can click **⋮ > Edit Config** again to check the newly added disk. Meanwhile, you can also add the "Host/Disk" tag (details are described in the [next section](#storage-tags)).
```

Author: updated!
